spec/javascript/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -311,6 +312,7 @@ describe('document-capture/components/document-capture', () => {
 
     context('in person steps', () => {
       it('renders the step indicator', async () => {
+        const callback = sinon.spy();
```

Comment:
Since we've set it up already in this file, I think we can use `sandbox` to avoid pulling in an extra dependency.

```suggestion
        const callback = sandbox.spy();
```